rpa-home/modules/ROOT/pages/anypoint-migration-overview.adoc

```diff
@@ -0,0 +1,41 @@
+= Anypoint Platform Migration Overview
+
+Starting July 1st, RPA Manager integrates with Anypoint Platform to unify the login experience and provide support for external identities (IDP).
+
+If your organization acquired the MuleSoft Automation Bundle after the integration, your account is not affected by the migration because your organization is already in Anypoint Platform. 
+
+The migration process for existing RPA customers starts on July 1st and is carried out automatically in different waves. To prepare for the migration, ensure you follow the <<pre-migration, Pre-migration Steps>>.
+
+For additional questions, contact us at rpa_migration_support@salesforce.com.
+
+== Summary of Changes
+
+After your organization is migrated to Anypoint Platform, the following changes apply:  
+
+* Users are required to reconfigure 2FA for their Anypoint Account.
```

Comment:
not entirely true. If the user already had an Anypoint Platform user with the same email, they will just use that.
Maybe we could reframe it to something like: users that didn't previously have an Anypoint Platform user with the same email address as in RPA, will be invited to join the Anypoint Platform organization. At this point, they will be required to enter a password to be used and to configure the 2FA.